kairos-session-code/deposit-session/src/main.rs

```diff
@@ -30,6 +31,7 @@ pub extern "C" fn call() {
         "deposit",
         runtime_args! {
             "temp_purse" => temp_purse,
+            "recipient" => public_key,
```

Comment:
Should this really be a Casper PublicKey? Shouldn't this be a Kairos PublicKey?

Reply:
Other than that lgtm.

Reply:
Kairos PublicKey is just Vec<u8> which would need CLType::Any, so I convert to bytes in the contract instead.